src/main/java/com/ifmo/aumetov/CalcMethod.java

```diff
@@ -0,0 +1,80 @@
+package com.ifmo.aumetov;
+
+import java.util.*;
+
+public class CalcMethod {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'CalcMethod' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4765332)